drizzlepac/haputils/catalog_utils.py

```diff
@@ -686,7 +686,8 @@ def verify_crthresh(self, n1_exposure_time):
                 n_sources = source_cat.sources_num_good  # len(source_cat)
                 all_sources = len(source_cat)
                 log.info("{} catalog with {} good sources out of {} total sources :  CR threshold = {}".format(cat_type, n_sources, all_sources, thresh))
-                if n_sources < thresh:
+                # n_sources == 0 should never get here, but just to cover the past case of 0 < 0.0
```

Comment:
> but just to cover the past case of 0 < 0.0

I am not sure I understand this. How can 0 < 0.0? Is `n_sources` a floating point number? Should it be a floating point number? And if it is, do you mean that it could be a negative number?

Reply:
What I see in the code is:

```python
source_cat.sources_num_good = len(np.where(crthresh_mask)[0])
```
on line 674 and on line 686 I see:

```python
n_sources = source_cat.sources_num_good # len(source_cat)
```

`n_sources` is an integer and I am not sure what case proposed changes deal with.

Reply:
In the past a bug was tripped here because n_sources (with a value of 0) was NOT less than thresh (with a value of 0.0).  This issue will probably not ever be tripped again as no sources being found is trapped in other places in the code.  However, to be sure, I wanted to put in an evaluation which would evaluate properly on 0, but not change previous results.

Reply:
Ah, it was `tresh` that was a floating point.